.github/workflows/discourse-plugin.yml

```diff
@@ -316,3 +329,24 @@ jobs:
         with:
           name: failed-system-test-screenshots
           path: tmp/capybara/*.png
+
+      - uses: LouisBrunner/checks-action@v1.6.1
+        name: Report optional check failure
+        if: |
+          matrix.updated_ember == true &&
+          (matrix.build_type == 'frontend' || matrix.build_type == 'system') &&
+          (steps.qunit.outcome == 'failure' || steps.system_spec.outcome == 'failure')
```

Comment:
do we need to check `build_type`? i.e. can qunit or system_spec steps have the failure outcome in other build_types?

Reply:
It seems to work, so they must have some kind of 'skipped' status on other runs